lib/src/jira/api.rs

```diff
@@ -407,6 +570,44 @@ fn parse_pending_version_field(field: &serde_json::Value) -> Vec<version::Versio
         .collect::<Vec<_>>()
 }
 
+/// Parse a Jira multi-select field that returns an array of option objects
+fn parse_multiselect_field(field: &serde_json::Value) -> Option<String> {
+    if field.is_null() {
+        None
+    } else if let Some(options_array) = field.as_array() {
+        let values: Vec<String> = options_array
+            .iter()
+            .filter_map(|option| {
+                option.get("value")
+                    .and_then(|v| v.as_str())
+                    .map(|s| s.to_string())
+            })
+            .collect();
+
+        if values.is_empty() {
+            None
+        } else {
+            Some(values.join(", "))
+        }
+    } else {
+        // Fallback: treat as simple string field
+        Some(field.as_str().unwrap_or("").to_string())
+    }
+}
+
+/// Parse a Jira single-select field that returns a single option object
+fn parse_singleselect_field(field: &serde_json::Value) -> Option<String> {
+    if field.is_null() {
+        None
+    } else if let Some(value_str) = field.get("value").and_then(|v| v.as_str()) {
+        // Single-select field: extract "value" from the option object
+        Some(value_str.to_string())
+    } else {
+        // Fallback: treat as simple string field
+        Some(field.as_str().unwrap_or("").to_string())
+    }
+}
+
```

Comment:
```suggestion
```

These are never used. There are some unit tests  that test these in isolation but it has no use in the production code that I can see.